visvis2/renderers/_wgpu.py

```diff
@@ -95,7 +121,7 @@ def compose_pipeline(self, wobject):
             ctypes.memmove(buffer.mapping, array.ctypes.data, nbytes)
             buffer.unmap()
             shader_location = len(buffers)
```

Comment:
perhaps unrelated but why is `shader_location = len(buffers)`?

Reply:
Looked it up in the spec:

> Each attribute appears as a separate input in a vertex shader, each bound by a numeric location, which is specified by shaderLocation.

So you're probably counting as you expand the array. Perhaps using `enumerate()` in the for loop would be a little more explicit?

Reply:
Nice catch. I think you're right and this should just enumerate. This is probably a leftover from copying this code. I have to get the namespaces (slot-spaces?) straight.